internal/venice-client-common/src/main/java/com/linkedin/venice/stats/VeniceOpenTelemetryMetricsRepository.java

```diff
@@ -154,6 +154,22 @@ public LongCounter getCounter(String name, String unit, String description) {
     });
   }
 
+  public Object getInstrument(MetricEntity metricEntity) {
+    switch (metricEntity.getMetricType()) {
+      case HISTOGRAM:
+        return getHistogram(metricEntity.getMetricName(), metricEntity.getUnit(), metricEntity.getDescription());
+      case HISTOGRAM_WITHOUT_BUCKETS:
+        return getHistogramWithoutBuckets(
+            metricEntity.getMetricName(),
+            metricEntity.getUnit(),
+            metricEntity.getDescription());
+      case COUNTER:
+        return getCounter(metricEntity.getMetricName(), metricEntity.getUnit(), metricEntity.getDescription());
+      default:
+        throw new VeniceException("Unknown metric type: " + metricEntity.getMetricType());
+    }
+  }
```

Comment:
Can we reduce the boilerplate by making these three functions take a single `MetricEntity` param rather than the three Strings?